GestureHandler.js

```diff
@@ -15,46 +13,19 @@ import {
   StyleSheet,
   FlatList,
   Platform,
+  processColor,
+  Touchable,
 } from 'react-native';
-import processColor from 'react-native/Libraries/StyleSheet/processColor';
-import Touchable from 'react-native/Libraries/Components/Touchable/Touchable';
 import deepEqual from 'fbjs/lib/areEqual';
 import PropTypes from 'prop-types';
 
+import GestureHandlerButton from './GestureHandlerButton';
+import GestureHandlerModule from './GestureHandlerModule';
 import gestureHandlerRootHOC from './gestureHandlerRootHOC';
 
-const RNGestureHandlerModule = NativeModules.RNGestureHandlerModule;
-
-/* Wrap JS responder calls and notify gesture handler manager */
-const { UIManager } = NativeModules;
-const {
-  setJSResponder: oldSetJSResponder,
-  clearJSResponder: oldClearJSResponder,
-} = UIManager;
-UIManager.setJSResponder = (tag, blockNativeResponder) => {
-  RNGestureHandlerModule.handleSetJSResponder(tag, blockNativeResponder);
-  oldSetJSResponder(tag, blockNativeResponder);
-};
-UIManager.clearJSResponder = () => {
-  RNGestureHandlerModule.handleClearJSResponder();
-  oldClearJSResponder();
-};
-
-// Add gesture specific events to genericDirectEventTypes object exported from UIManager
-// native module.
-// Once new event types are registered with react it is possible to dispatch these
-// events to all kind of native views.
-UIManager.genericDirectEventTypes = {
-  ...UIManager.genericDirectEventTypes,
-  onGestureHandlerEvent: { registrationName: 'onGestureHandlerEvent' },
-  onGestureHandlerStateChange: {
-    registrationName: 'onGestureHandlerStateChange',
-  },
-};
-
-const State = RNGestureHandlerModule.State;
-
-const Directions = RNGestureHandlerModule.Direction;
+import Directions from './Directions';
```

Comment:
We used to take directions list from `RNGestureHandlerModule` can we have a separate implementation for `Directions.web.js` to keep it the same way on mobile platforms?

Reply:
Yep, it's implemented now.